examples/ordering/run.sh

```diff
@@ -1,97 +1,187 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-usage() {
-    echo "Usage: ./run.sh [-n N_NODES] [-m N_MALFUNCTIONING_NODES] [-s N_STALLING_DATA_PROVIDERS] [-c N_CRASHES] [-o N_ORDERED_PER_CRASH] [-d RESTART_DELAY]"
+set -eou pipefail
+
+function usage() {
+  cat << EOF
+Usage:
+  This script is a demonstration usage of AlephBFT protocol, in which there are N nodes and they want to achieve
+  a consensus with regards to provided data. The data sent to AlephBFT from each node is a stream of integers from range
+  [0, N * DATA_SIZE_BATCH), where DATA_SIZE_BATCH is configurable. Each node of index 'i'  sends to the consensus
+  integers from range [i * DATA_SIZE_BATCH; (i + 1) * DATA_SIZE_BATCH). where 0 <= i < N. At the end, each node makes
+  sure that it receives all integers from range [0, N * DATA_SIZE_BATCH), each integer exactly once.
+
+  N nodes are started on your machine, and they communicate via UDP. Not all nodes behave correctly - some of them crash
+  or are stuck while providing data.
+
+  This script is using aleph-bft-examples-ordering and assumes to be available in a relative folder from this script path
+    ../../target/release/aleph-bft-examples-ordering
+
+   $0
+     [-n|--nodes NODES]
+       number of all non-crashing nodes; some of them can have stalled data provider
+     [-c|--crashing-nodes CRASHING_NODES]
+       number of nodes that crash while providing data
+     [-s|--stalling-data-providers STALLING_DATA_PROVIDERS]
+       number of nodes that eventually stall while providing data; must be less than --nodes
+     [--crashes-count CRASHES_COUNT]
+       how many times a crashing node should crash
+     [--data-items DATA_ITEMS]
```

Comment:
Make this consistent with `DATA_SIZE_BATCH` above (no opinion as to which name is better).

Reply:
Done